app/views/insured/families/_inbox.html.erb

```diff
@@ -2,39 +2,32 @@
   <div class="row">
     <div class="inboxes" id="inbox_form">
       <div class="pull-right">
-        <%= link_to(inbox_insured_families_path(@provider, folder: 'Deleted'), class: 'pull-right') do %>
-        <% if @folder == 'Deleted' %>
-        <strong>
-          <%= l10n("deleted") %>
-        </strong>
-        <% else %>
-          <%= l10n("deleted") %>
-        <% end %>
-        <% end %>
-        <span class="pull-right">
-          &nbsp;|&nbsp;
-        </span>
-        <%= link_to(inbox_insured_families_path(@provider), class: 'pull-right') do %>
-        <% if @folder == 'Inbox' %>
-        <strong>
-          <%= l10n("inbox") %>
-        </strong>
-        <% else %>
-          <%= l10n("inbox") %>
-        <% end %>
-        <span class="badge message-badge"><%= @provider.inbox.unread_messages.size %></span>
-        <% end %>
-        <%- if @sent_box == true %>
-        <%= link_to(inbox_insured_families_path(@provider, folder: 'Sent')) do%>
-        <% if @folder == 'Sent' %>
-        <strong>
-          <%= l10n("sent") %>          
-        </strong>
-        <% else %>
-          <%= l10n("sent") %>
-        <% end %>
-        <% end %>
+        <a href="<%= inbox_insured_families_path(@provider) %>">
+          <% if @folder == 'Inbox' %>
+            <strong><%= l10n("inbox") %></strong>
+          <% else %>
+            <%= l10n("inbox") %>
+          <% end %>
+          <span class="badge message-badge"><%= @provider.inbox.unread_messages.size %></span>
+        </a>
+        <% if @sent_box == true %>
+          <span>&nbsp;|&nbsp;</span>
+          <a href="<%= inbox_insured_families_path(@provider, folder: 'Sent') %>">
+            <% if @folder == 'Sent' %>
+              <strong><%= l10n("sent") %></strong>
+            <% else %>
+              <%= l10n("sent") %>
+            <% end %>
+          </a>
         <% end %>
+        <span>&nbsp;|&nbsp;</span>
```

Comment:
any chance we could get rid of the nbsp's?

Reply:
https://github.com/ideacrew/enroll/pull/3479/commits/f4a4d2d02791627c19d5224d34def741accbaf94